osu.Game/Skinning/Editor/SkinSelectionHandler.cs

```diff
@@ -151,11 +232,24 @@ public override bool HandleMovement(MoveSelectionEvent<ISkinnableDrawable> moveE
             {
                 Drawable drawable = (Drawable)c.Item;
                 drawable.Position += drawable.ScreenSpaceDeltaToParentSpace(moveEvent.ScreenSpaceDelta);
+
+                updateDrawableAnchorIfUsingClosest(drawable);
             }
 
             return true;
         }
 
+        private void updateDrawableAnchorIfUsingClosest(Drawable drawable)
+        {
+            if (drawable is ISkinnableDrawable { OverridesClosestAnchor: true }) return;
```

Comment:
Is pattern matching like this acceptable?

Reply:
No.